src/main/resources/dev/definitions/prisons/orphanage/ors-alerts-report.json

```diff
@@ -23,8 +23,45 @@
+      "parameters": [
+        {
+          "index": 0,
+          "name": "establishment_code",
+          "filterType": "text",
```

Comment:
If the "specialType": "establishment_code"  (and "specialType""wing) is included, along with "filterType":"autocomplete" then the user will be able to search the list of establishments.

Reply:
Update: This will not work yet as its pending some FE work.
So it looks good to me as it is.